internal/libs/testhelpers/common.go

```diff
@@ -30,3 +32,10 @@ func CreateOracleIDToP2pID(ids ...int) map[commontypes.OracleID]libocrtypes.Peer
 	}
 	return res
 }
+
+func ExtractSequenceNumbers(outcome exectypes.Outcome) []ccipocr3.SeqNum {
+	sequenceNumbers := slicelib.Map(outcome.Report.ChainReports[0].Messages, func(m ccipocr3.Message) ccipocr3.SeqNum {
+		return m.Header.SequenceNumber
+	})
+	return sequenceNumbers
+}
```

Comment:
I don't think this should depend on the execute package. It's only used in one place so maybe leave the original code?

Reply:
Moved to test_utils.go